.github/workflows/documentation.yaml

```diff
@@ -1,4 +1,4 @@
-name: Workflow to deploy documentation for SeqAn3
+name: Deploy https://docs.seqan.de/
```

Comment:
```suggestion
name: Deploy SeqAn3 documentation to https://docs.seqan.de/
```

Reply:
See https://github.com/seqan/seqan3/pull/1816#issuecomment-630074757

enrico asked me to shorten the name

Reply:
But I don't deploy the server? Then rather have it `Deploy documentation (https://docs.seqan.de/)`

Reply:
afaik; we deploy the sources to the server